tensorboard/webapp/widgets/histogram/histogram_util_test.ts

```diff
@@ -347,26 +326,25 @@ describe('histogram util', () => {
       });
 
       it(
-        'produces result bins with full and partial contributions from ' +
-          'multiple 0 width bins',
+        'produces result bins from multiple 0 width bins in different ' +
+          'steps',
```

Comment:
To simulate different steps, it would be best if for the input here we call `binsToHistogram()` more than once, e.g. something like this:

```ts
          expect(
            histogramsToBins(
              buildNormalizedHistograms(
                [
                  binsToHistogram([{x: 0, dx: 0, y: 200}]),
                  binsToHistogram([{x: 1.0, dx: 0, y: 100}]),
                ],
                2
              )
            )
          ).toEqual([...]);
```

That's because each call of `binsToHistogram()` is meant to represent a single step of data, and in a single step of the valid v3 format, there shouldn't be more than one zero-width bin that has data in it.

The resulting output should be the same; this is just a more accurate representation of how the buildNormalizedHistograms() API is intended to be used.

Reply:
If using two `binsToHistogram` above the result becomes:
```
[
  {x: 0, dx: 0.5, y: 200},
  {x: 0.5, dx: 0.5, y: 0},
],
[
  {x: 0, dx: 0.5, y: 0},
  {x: 0.5, dx: 0.5, y: 100},
]
```
hence moving this case under the `multiple histograms` category below.

Reply:
Got it, ok, that makes sense!